### ISSUE TITLE
Unable to create text below accordion

### ISSUE THREAD

**Issue by Graham-McCracken** [tinymce/tinymce#8890]
The accordion plugin is a welcome new feature. 

If it is the final object in a document, I am unable to navigate below it (clicking below/ keyboard right/down arrow/ enter does not work)

**Comment by GBRielGCLC** [comment]
For me doesn't work anyhere, even in the [documentation page](https://www.tiny.cloud/docs/tinymce/6/accordion/), it's like the arrow insn't clickable anymore.

**Comment by MrMooky** [comment]
The only way to jump "out" of the accordion is by tapping enter when the cursor is at the end of the accordion. Let's take the [demo](https://www.tiny.cloud/docs/tinymce/6/accordion/) as an example. When you open "_Are there other Accordion options?_" and click after "_See below for details_" and then hit enter twice, you are not within the accordion anymore.

Stumbled upon this by accident. I'd argue that most of my users will not understand that.

> ... it's like the arrow insn't clickable anymore

@GBRielGCLC The arrow is not clickable in the RTE itself. You will need to use the toggle icon for this. At least that is how I understand it.

![Screenshot 2023-07-14 at 14 45 19](https://github.com/tinymce/tinymce/assets/3603346/78c45949-987a-4a26-b1f8-9321b785c7b6)

**Comment by Graham-McCracken** [comment]
Agree, I believe most of my users would not understand it either. I tried for several minutes to navigate below it when it's the final line of text in a doc - without success. I'll have a look at the "enter" option. 

But I do think the user experience could be easily refined.

**Comment by TheSpyder** [comment]
Yes, we hope to iterate on the accordion experience in the coming releases. The arrow, for example, is a pseudo-element and it's not easy to detect clicks on it. Browsers don't toggle on click when ContentEditable is enabled.

For adding content above and below, it is something we're aware of. Accordions are like tables, in that respect, but we only implemented newlines on enter for this release. Not when using the arrow keys.

**Comment by alpt-cmsvieira** [comment]
Hi,
there's an update on this issue? 

regards
